### DISCUSSION TITLE
lag and stutter

### DISCUSSION THREAD

**Discussion by Jadshehadeh** [saadeghi/daisyui#1120] [Q&A] [answered]
https://user-images.githubusercontent.com/43533643/189502790-61eadd03-8149-42f5-89b5-3ddd647cc92f.mp4

Hello,

Does anyone know how to fix this lag ? 

thanks.

**Comment by saadeghi** [comment, marked as answer]
- It might get better if you close the dev tools. When inspector is open, browser has to do extra jobs on top of rendering actual page content.
- rendering blur effect is considered as a heavy task for browsers so (if possible) try disabling the blur to see if it gets better.
- If you have other heavy graphics on the page, that might be the reason. I think you will experience a similar lag with [Stripe's homepage](https://stripe.com/) for reference.

**Reply by Jadshehadeh** [reply]
thanks for replying. 
I tried everything you mentioned but it did nothing because the problem was with the inline tailwind css class "transition-all duration-300".
When I made the same transition in the global.css like this: 
.cart{
  transition: transform .3s ease;
}
.cart:hover{
  transform: scale(1.08,1.08);
}

the problem is solved and it wasn't lagging anymore 
I don't know why.
Thank you again.
